esmvaltool/cmorizers/obs/cmorize_obs_era5.py

```diff
@@ -0,0 +1,156 @@
+"""ESMValTool CMORizer for ERA5 data.
+
+Tier
+    Tier 3: restricted datasets (i.e., dataset which requires a registration
+ to be retrieved or provided upon request to the respective contact or PI).
+
+Source
+    https://cds.climate.copernicus.eu/cdsapp#!/dataset/reanalysis-era5-pressure-levels
+    https://cds.climate.copernicus.eu/cdsapp#!/dataset/reanalysis-era5-single-levels
+
+Last access
+    20190718
+
+Download and processing instructions
+    This cmorization script currently supports hourly data of the following
+variables:
+        2m_temperature
+        potential_evaporation
+        total_precipitation
+
+    Downloading ERA5 data can either be done via the Climate Data Store (cds)
+web form or era5cli:
+        $pip install era5cli
+        $era5cli hourly --variables total_precipitation --startyear 1990
```

Comment:
I would suggest to add the link to the [era5cli documentation](https://era5cli.readthedocs.io/en/latest/), at least for the chapter on how to configure the key ascii file.

Reply:
@hb326 the era5clim python package might be useful for you!

Reply:
Also, why not adding this package to our setup?

Reply:
There is no conda package available, so for now I think it might be best not to add it. Do you think it would be useful for many users? If so, we could add it in the future (and work on getting it in conda), to make obtaining at least some observational datasets easier. Shall I make an issue?

Reply:
> Do you think it would be useful for many users?

It depends on how many users need to download ERA5 data, but for them is definitely useful.
Downloading the data "manually" is not an option.

> Shall I make an issue?

Go ahead! 
Would it also work for ERA-Interim data?

Reply:
No, I think it was specifically designed for ERA5. However, it just adds some command line convenience to the CDS Python API, which allows downloading any dataset in the Copernicus Climate Data Store, see https://pypi.org/project/cdsapi/

Reply:
There is an analogous one for ECMWF data (https://pypi.org/project/ecmwf-api-client/), but it does not probably make sense to invest much resources on ERA-Interim now, since it has been replaced by ERA5.

Reply:
Issue created https://github.com/ESMValGroup/ESMValTool/issues/1311